docker.config.pisces

```diff
@@ -0,0 +1,18 @@
+%% -*- erlang -*-
+[
+  "config/sys.config",
+  {lager,
+    [
+      {log_root, "/var/log/miner"}
+    ]},
+    {blockchain,
+    [
+      {key, {ecc, [{key_slot, 0}, {onboarding_key_slot, 0}, {bus, "i2c-0"}, {address, 16#60}]}}
```

Comment:
`{address, 16#60}` is the [default](https://github.com/helium/miner/blob/80ac550710172665ea71f7a0e8f1f8260a643b95/src/miner_keys.erl#L145)? Noting that we don't set it anywhere else.

Reply:
@marvinmarnold good point - i will remove it